OpenCL_C.txt

```diff
@@ -3299,6 +3357,10 @@ The macro names defined by the C99 specification but not currently supported
 by OpenCL are reserved for future use.
 
 The predefined identifier `+__func__+` is available.
+
+In OpenCL C 3.0 there are a number of optional predefined macro indicating
+optional language features. Such macros are listed in
+<<table-optional-lang-features>>.
```

Comment:
```suggestion
optional language features. Such macros are listed in the
<<table-optional-lang-features, optional features in OpenCL C 3.0 table>>.
```

Not strictly required, but as is it ends up with a `..` at the end of the sentence.